<ui version="4.0" >
 <class>IdentityStatusBase</class>
 <widget class="QWidget" name="IdentityStatusBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>146</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="identityName" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Identity Name</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item rowspan="2" row="0" column="0" >
      <widget class="QLabel" name="photo" >
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="text" >
        <string>&lt;a href="identity::getavatar">No Photo&lt;/a></string>
       </property>
       <property name="textFormat" >
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="nickLabel" >
         <property name="text" >
          <string>Nick:</string>
         </property>
         <property name="buddy" >
          <cstring>nickName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nickName" />
       </item>
      </layout>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="identityStatus" >
       <property name="text" >
        <string>&lt;a href="identity:status">&lt;img src=identityStatus> Online&lt;/a></string>
       </property>
       <property name="textFormat" >
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents" >
        <bool>false</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="accounts" >
     <property name="text" >
      <string>&lt;img src="account"> Account ID (&lt;a href="account:onlineStatus">Away&lt;/a>)</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
